src/helpers/uttu.ts

```diff
@@ -0,0 +1,49 @@
+import messages, {
+  UttuCode,
+  UttuSubCode,
+  CombinedUttuCode
+} from './uttu.messages';
+import { IntlFormatters } from 'react-intl';
+
+type Extensions = {
+  code: UttuCode;
+  subCode: UttuSubCode;
+  metaData: { [key: string]: object };
+};
+
+type UttuError = {
+  response:
```

Comment:
Kanskje penere med `response?: { ....}`  i stedet for pipe?

Reply:
Enig i at det er penere, men i teorien så er det litt forskjell på `key?: value` og `key: undefined | value`. Førstnevnte sier at `key` ikke nødvendigvis finnes, mens sistnevnte sier at `key` er en required prop som kan ha verdi lik `undefined`. I praksis oppfører de seg dog likt, men jeg mener at sistnevnte er det som er rett semantisk sett :p